src/theme/index.js

```diff
@@ -32,6 +34,19 @@ const flexRowNoWrap = css`
 const white = '#FFFFFF'
 const black = '#000000'
 
+export default function ThemeProvider({ children }) {
```

Comment:
hm do we want to do this here? doesn't it make sense to just rely on the setters in each of the individual pages?